pennylane/transforms/mitigate.py

```diff
@@ -446,6 +451,7 @@ def mitigate_with_zne(
             folding=fold_global,
             extrapolate=poly_extrapolate,
             extrapolate_kwargs={'order': 2})
+        @partial(qml.compile, basis_set = ["RY", "CZ"])
```

Comment:
With PL dev, we'd do

```
@partial(qml.decompose, gate_set ={"RX", "RZ"})
```

See https://github.com/PennyLaneAI/pennylane/pull/6334

Reply:
Ahh, I see, it is still not merged yet. That's why couldn't find it lol. I'll wait until it gets merged I suppose.

Reply:
It's close to being merged!

Reply:
Merged!